### PR TITLE
Tk friends list

### PR DESCRIPTION
**Pull request by thk421**
The user blocking system has been added

### CODE REVIEW

**Review by cymnyon — DISMISSED** [review]

**Review by oshley213 — DISMISSED** [review]
good

**Review by naraeleee — DISMISSED** [review]
good

File: src/main/java/pages/FriendsPage.java [hunk 1]
```diff
@@ -29,6 +32,9 @@ public Page run() {
             case BACK_HOME:
                 redirect = this.router.getHomePage();
                 System.out.println("Redirecting to Home Page...");
```
Comment (jeffhcs) [inline src/main/java/pages/FriendsPage.java:34]: This is missing a break statement.

**Review by jeffhcs — CHANGES_REQUESTED** [review]

File: src/main/java/pages/FriendsPage.java (hunk 1 above)
Comment (thk421) [reply src/main/java/pages/FriendsPage.java:34]: okay

**Review by jeffhcs — APPROVED** [review]
Looks good now, nice work!

**Review by oshley213 — APPROVED** [review]
very good

**Review by cymnyon — APPROVED** [review]

**Pull request merged**